### ISSUE TITLE
Automate the script with Github Actions

### ISSUE THREAD

**Issue by Zaid-maker** [jmfayard/timetraveler#1]
Love to see if we can try to add a github Action to run the script every 10 minutes or so!

**Comment by jmfayard** [comment]
Good idea but re-running wouldn't bring much, my vanity metrics are already all green after one run.

One thing you could play with is to create a GitHub Action so that someone new to the project can create all those commits on GitHub actions instead of locally

I'm not gonna do it, but anyone willing to toy with it, please have fun :)

PS: a fun side-project of mine
https://github.com/typesafegithub/github-workflows-kt

**Comment by Zaid-maker** [comment]
I will have a look at your project 👏
